lib/class/xml_data.class.php

```diff
@@ -77,23 +77,6 @@ public static function set_limit($limit)
     } // set_limit
 
     /**
-     * set_type
-     *
-     * This sets the type of XML_Data we are working on
-     *
-     * @param    string    $type    XML_Data type
-     * @return    void
-     */
-    public static function set_type($type)
-    {
-        if (!in_array($type,array('rss','xspf','itunes'))) {
-            return false;
-        }
-
-        self::$type = $type;
-    } // set_type
-
-    /**
```

Comment:
Why was it deleted? Isn't it used? If this is the case, this may be a bug elsewhere.